MonMaraicher/MapView.swift

```diff
@@ -10,9 +10,32 @@ import MapKit
 
 struct MapView: View {
     
+    @StateObject private var viewModel = MapViewModel()
+    
     var body: some View {
-        
-        Text("Hello world!")
+        Map(position: $viewModel.userPosition, selection: $viewModel.selectedFarmerPlace) {
+            
+            ForEach(viewModel.allFarmerPlaces, id: \.id) { place in
+                Marker(place.name, coordinate: CLLocationCoordinate2D(latitude: place.location.latitude, longitude: place.location.longitude))
+                    .tag(place)
+                    .tint(.green)
+            }
+            
+            UserAnnotation()
+            
+        }
+        .onChange(of: viewModel.selectedFarmerPlace, { oldValue, newValue in
```

Comment:
Suggestion: I always give a clear naming for closure parameters:
```.onChange(of: viewModel.selectedFarmerPlace, { _, newSelectedFarmerPlace in```

Reply:
It's clearer. New value and old value were the default values of .onChange